frontend/public/components/dashboard/dashboards-page/overview-dashboard/inventory-card.tsx

```diff
@@ -73,13 +73,15 @@ const ClusterInventoryItem = withDashboardResources(
       let additionalResourcesLoadError = false;
       if (additionalResources) {
         additionalResourcesLoaded = additionalResources.every((r) =>
-          _.get(resources[r.prop], 'loaded'),
+          // If it's optional, ignore it for the `loaded` indicator
+          r.optional ? true : _.get(resources[r.prop], 'loaded'),
         );
         additionalResources.forEach((r) => {
           additionalResourcesData[r.prop] = _.get(resources[r.prop], 'data');
         });
         additionalResourcesLoadError = additionalResources.some(
```

Comment:
you can `filter` the optional resources first `additionalResources.filter((r) => !r.optional).some(....` - it will be easier to understand then